enhancements/accelerators/out-of-tree-drivers.md

```diff
@@ -0,0 +1,418 @@
+---
+title: Supporting out-of-tree drivers on BareMetal & OpenShift
+  - "@zvonkok"
+reviewers:
+  - TBD
+  - "@mrunalp"
+approvers:
+  - TBD
+  - "@mrunalp"
+creation-date: 2020-04-03
+last-updated: 2020-06-02
+status: provisional
+see-also:
+  - "/enhancements/TODO.md"  
+replaces:
+  - "/enhancements/TODO.md"
+superseded-by:
+  - "/enhancements/TODO.md"
+---
+
+# Supporting out-of-tree drivers on BareMetal & OpenShift
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+1. Are users willing to provide precompiled DriverContainers (We have already some commitments)?  This helps immensely in disconnected environments and only validated combinations of driver + kernel would be e.g. provided. Source instal should be the fallback solution
+
+## Summary
+
+Support out-of-tree drivers on OpenShift for enabling custom hardware and third-party kernel modules on container and non-container based operating systems. 
+
+## Terminology
+### DriverContainers
+
+DriverContainers are used more and more in cloud-native environments, especially  when run on pure container operating systems to deliver hardware drivers to the host.
+
+DriverContainers are not only a delivery mechanism. They are far more than that. They enable the complete user and kernel stack. They configure the hardware, start daemons that are essential for hardware to work and perform other important tasks. 
```

Comment:
> DriverContainers are not only a delivery mechanism. They are far more than that. 
DriverContainers are not only a delivery mechanism; they are far more than that.